src/Public.Api/StreetName/BackOffice/StreetNameBackOfficerController-CorrectRetirement.cs

```diff
@@ -0,0 +1,91 @@
+namespace Public.Api.StreetName.BackOffice
+{
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Be.Vlaanderen.Basisregisters.Api.Exceptions;
+    using Common.Infrastructure;
+    using Infrastructure;
+    using Infrastructure.Swagger;
+    using Microsoft.AspNetCore.Http;
+    using Microsoft.AspNetCore.Mvc;
+    using Microsoft.AspNetCore.Mvc.Infrastructure;
+    using RestSharp;
+    using StreetNameRegistry.Api.Legacy.StreetName.Responses;
+    using Swashbuckle.AspNetCore.Annotations;
+    using Swashbuckle.AspNetCore.Filters;
+    using ProblemDetails = Be.Vlaanderen.Basisregisters.BasicApiProblem.ProblemDetails;
+
+    public partial class StreetNameBackOfficeController
+    {
+        /// <summary>
+        /// Corrigeer de opheffing van een straatnaam.
+        /// </summary>
+        /// <param name="objectId">Identificator van de straatnaam.</param>
+        /// <param name="actionContextAccessor"></param>
+        /// <param name="problemDetailsHelper"></param>
+        /// <param name="correctStreetNameRetirementToggle"></param>
+        /// <param name="ifMatch">If-Match header met ETag van de laatst gekende versie van de straatnaam (optioneel).</param>
+        /// <param name="cancellationToken"></param>
+        /// <response code="202">Als het ticket succesvol is aangemaakt..</response>
+        /// <response code="400">Als uw verzoek foutieve data bevat.</response>
+        /// <response code="404">Als de straatnaam niet gevonden kan worden.</response>
+        /// <response code="406">Als het gevraagde formaat niet beschikbaar is.</response>
+        /// <response code="412">Als de If-Match header niet overeenkomt met de laatste ETag.</response>
+        /// <response code="429">Als het aantal requests per seconde de limiet overschreven heeft.</response>
+        /// <response code="500">Als er een interne fout is opgetreden.</response>
+        /// <returns></returns>
+        [ApiOrder(ApiOrder.StreetName.Edit + 4)]
```

Comment:
Should this be + 8? Correct approval is + 6. Correct rejection will be 7. Then this should be +8, no?